doc/decisions/frontend_timeline_concept.md

```diff
@@ -0,0 +1,25 @@
+# Timeline: Custom Date Picker Implementation Decision
+
+## Problem Statement
+
+To implement the timeline feature, we require a custom date picker capable of displaying indicators for plant additions and removals.
+
+## Proposed Solutions
+
+Initially, the team considered using [ReactCharts](https://react-charts.tanstack.com/), a powerful charting library. However, it is not designed for date selection. As a result, we have decided to create custom components for building the timeslider:
+
+- **Picker Slider:** A draggable and scrollable slider where the middle element is the selected one. We will explore existing solutions that may meet our requirements. The elements displayed on the slider can also be customized to accommodate event indicators for each date.
```

Comment:
>  We will explore existing solutions that may meet our requirements.

This should happen in this decision, the requirements are the 2.nd and 3.rd point in the decision template.